deploy/examples/subvolumegroup.yaml

```diff
@@ -9,3 +9,8 @@ spec:
   # name: group-a
   # filesystemName is the metadata name of the CephFilesystem CR where the subvolume group will be created
   filesystemName: myfs
+  # reference https://docs.ceph.com/en/quincy/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups
+  # pinning:
+  #   # by default set to true with default values(type=distributed,setting=1)
+  #   type: distributed   # Values: random;export;distributed
+  #   setting: 1          # Values: export=<0-256>(disabled=-1), distributed=<0, 1>(disabled=0), random=[0.0, 1.0](disabled=0.0)
```

Comment:
For validating settings value I can open a separate PR once the discussion is completed